### ISSUE TITLE
Add text field to link (e.g. external link) for the event

### ISSUE THREAD

**Issue by Hobokens** [derhansen/sf_event_mgt#1118]
### Extension version
6.4

### Short problem description
I want to add an external link to an event. In frontend, I get the label 'Link' and the link is named link, too. I would prefer to 
name the link by myself.

### Steps to reproduce
Add a link to an event under the tab additional

### Server configuration
**TYPO3 version:**
11.5.25

**PHP version:**
8.1.16

**Comment by derhansen** [comment]
Note, that the included templates in the extension are just for demonstration purposes only and should always be changed by overriding the original templates.

If you want to link the name by yourself, you can use TYPO3s default link functionality as shown below:

![image](https://user-images.githubusercontent.com/2629896/236887302-1d3ff964-f31b-42d9-bd5e-4c6b0d5365ae.png)

In your custom template, you can then use the `typolink` ViewHelper to output the link title as shown below:

```
<f:link.typolink parameter="{event.link}" parts-as="linkParts">
    {linkParts.title}
</f:link.typolink>

```

The `parts-as` attribute resolves all typolink-parameters for usage in the link. This feature is available since TYPO3 10.4

**Comment by Hobokens** [comment]
Great! I didn't know that.
Thanx and have a nice evening.
